src/components/AddTask.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { ReactComponent as DeleteIcon } from 'assets/delete.svg';
+import { useDispatch } from 'react-redux';
+import { addTodo, setIsAddingTask } from 'redux/actions/TodoAction';
+import { ENTER, ERROR } from 'utils/constants';
```

Comment:
Could we add context as prefix for every constants?
```suggestion
import { KEY_ENTER, RESPONSE_ERROR } from 'utils/constants';
```

Reply:
constants renamed